### ISSUE TITLE
Flash mode missing

### ISSUE THREAD

**Issue by ndoulgeridis** [wuwentao/midea_ac_lan#419]
### Device type and model (or SN)

> The model can be found in Settings -> Devices & Services -> Midea AC LAN -> Devices -> Device Info> The SN can be found in Settings -> Devices & Services -> Midea AC LAN -> Add device -> Just list appliances> The SN also can be found in your SmartHome APP

### Device type and model (or SN)

00000Q1B (8) 01JEVWJCXXXXXXXXXX

### The description of new feature

The model I own (Brezeeless) it has a feature called Flash.
This is different than boost. Boost is another option less efficient. The Flash is really the fastest but most power consuming way to cool or heat within some minutes your place.

However there is no way to find this Flash option in the home assistant. It is not an option in the card neither on the state have found something relevant to it.

Here is the screenshot in the app of the Flash feature:

![1734200319227](https://github.com/user-attachments/assets/574bca75-a706-4fb7-8995-287a21fc7315)

Here is the extra boost mode which is working from home assistant but is not the same with Flash:

![1734200319246](https://github.com/user-attachments/assets/70abc3ca-dc72-449a-8012-9fa889076e1a)

Is there way to add this attribute as well somehow? 

Thanks a lot!

**Comment by wuwentao** [comment]
you device type should be AC? and using Smarthome APP?
please provide your device SN, it should be available in your SmartHome app with device detail.

**Comment by ndoulgeridis** [comment]
Yes it's Air condition and I do use Smarthome APP and it's a "Brezeeless E Series CB1-18HRFN8-I"

When i select device and then select About device there is no AC but only terminal bardcode.

From Home assistant when I download statistics I get on the filename this, so maybe that's the SN (?): 01JEVW7EEHRHN68Q0XXXXXXXXX

**Comment by wuwentao** [comment]
@ndoulgeridis 

midea have more differnt types SN, used for different purpose.

1. below is a simple step: 
just backup your device xxx.json or rename it.
remove your device, enable debug log, add it with discovery, you can found the SN info in debug log.
for example:

```
2024-12-15 00:26:55.828 DEBUG (MainThread) [midealocal.discover] Found a supported device: {'device_id': xxx, 'type': 172, 'ip_address': '192.168.1.22', 'port': 6444, 'model': 'xxx', 'sn': 'xxx', 'protocol': 3}
```

this SN is what we expected.


method 3:  in addition, if you can login to HAOS and start docker.
also can be do below manual steps:

1. ssh login to HAOS
2. run docker with `docker exec -it homeassistant /bin/bash`
3. run discover debug command: `python3 -m midealocal.cli discover -d --host x.x.x.x` (x.x.x.x is your device IP)

you can also found the same output as debug log.

we can use your device SN to confirm your device protocol details.

**Comment by ndoulgeridis-tg** [comment]
OK with method 3 I think figured it out:

```
2024-12-17 14:04:39.745 DEBUG (MainThread) [midealocal.discover] Found a supported device: {'device_id': 152832116661925, 'type': 172, 'ip_address': '192.168.1.25', 'port': 6444, 'model': '00000Q11', 'sn': '000000P0000000Q1102C8D35DXXXXXXXX', 'protocol': 3}
```

**Comment by tastyllama** [comment]
Also looking for Flashcool (is that the same?) on a window unit.  The boost sends a command and it beeps, but does not seem to do anything.

Model: MAW10U1QWT

**Comment by tastyllama** [comment]
MAW10U1QWT has a dedicated button on the front, "FlashCool" which none of the buttons in the integration trigger.  I have tried "Boost Mode", "Breezeless", and "Comfort Mode" in addition to some of the others, and none trigger the "FlashCool".  Please let me know how to trigger this mode.
